corehq/toggles/__init__.py

```diff
@@ -1714,8 +1714,8 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 PREVENT_MOBILE_UCR_SYNC = StaticToggle(
     'prevent_mobile_ucr_sync',
-    'ICDS: Used for ICDS emergencies when UCR sync is killing the DB',
-    TAG_CUSTOM,
+    'Used for emergencies when UCR sync is causing operational problems',
```

Comment:
```suggestion
    'Prevent Mobile UCR sync (for when a UCR sync is causing operational problems)',
```
When I was reviewing yesterday, I thought this was the description. It's the name that shows up in the list, and everything else uses an imperative verb phrase, so I'm updating to match that